package.json

```diff
@@ -13,6 +13,22 @@
     "start:dev": "nest start --watch",
     "start:debug": "nest start --debug --watch",
     "start:prod": "node dist/main",
+    "typeorm": "ts-node -r tsconfig-paths/register -r ts-node/register ./node_modules/typeorm/cli.js",
+    "prod-migration:generate": "NODE_ENV=production npx typeorm migration:generate ./src/database/migrations/$TableName -d ./dist/config/datasource.js",
+    "local-migration:generate": "NODE_ENV=development ts-node -r tsconfig-paths/register ./node_modules/typeorm/cli.js migration:generate ./src/database/migrations/$TableName -d ./src/config/datasource.ts",
+    "migration:create": "NODE_ENV=production yarn run typeorm migration:create ./src/database/migrations/$TableName",
+    "local-migration:run": "NODE_ENV=dev ts-node ./node_modules/typeorm/cli.js migration:run -d ./src/config/datasource.ts",
+    "prod-migration:run": "NODE_ENV=production ts-node ./node_modules/typeorm/cli.js migration:run -d ./dist/config/datasource.js",
+    "local-migration:revert": "NODE_ENV=dev ts-node ./node_modules/typeorm/cli.js migration:revert -d ./src/config/datasource.ts",
+    "prod-migration:revert": "NODE_ENV=production ts-node ./node_modules/typeorm/cli.js migration:revert -d ./dist/config/datasource.js",
+    "local-migration:show": "NODE_ENV=dev ts-node ./node_modules/typeorm/cli.js migration:show -d ./src/config/datasource.ts",
+    "prod-migration:show": "NODE_ENV=production ts-node ./node_modules/typeorm/cli.js migration:show -d ./dist/config/datasource.js",
+    "local-migration:schema-sync": "NODE_ENV=dev ts-node ./node_modules/typeorm/cli.js schema:sync -d ./src/config/datasource.ts",
+    "prod-migration:schema-sync": "NODE_ENV=production ts-node ./node_modules/typeorm/cli.js schema:sync -d ./dist/config/datasource.js",
+    "local-migration:schema-log": "NODE_ENV=dev ts-node ./node_modules/typeorm/cli.js schema:log -d ./src/config/datasource.ts",
+    "prod-migration:schema-log": "NODE_ENV=production ts-node ./node_modules/typeorm/cli.js schema:log -d ./dist/config/datasource.js",
+    "local-migration:schema-drop": "NODE_ENV=dev ts-node ./node_modules/typeorm/cli.js schema:drop -d ./src/config/datasource.ts",
+    "prod-migration:schema-drop": "NODE_ENV=production ts-node ./node_modules/typeorm/cli.js schema:drop -d ./dist/config/datasource.js",
```

Comment:
Can we reduce the size of scripts and keep only the important ones?

Reply:
yup, sure

Reply:
@MHassanTariq have a look now.